test/deploy/validate-configs.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
```

Comment:
I only have a vague knowledge of bash scripts so can't really comment on the contents. @rubensworks or @RubenVerborgh are better suited.